recipes/libbsd/all/conanfile.py

```diff
@@ -21,62 +28,63 @@ class LibBsdConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    exports_sources = "patches/**"
-
-    _autotools = None
 
     def build_requirements(self):
         self.build_requires("libtool/2.4.6")
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        # plain C project
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+    
+    def generate(self):
+        if not cross_building(self):
+            env = VirtualRunEnv(self)
+            env.generate(scope="build")
+        tc = AutotoolsToolchain(self)
+        if is_apple_os(self):
+            tc.extra_cflags.append("-Wno-error=implicit-function-declaration")
```

Comment:
this cflag is not used correctly. @prince-chrismc any idea what I'm doing wrong?

Reply:
It is somehow related to the autoreconf() call

Reply:
Done